protege-editor-owl/src/main/java/org/protege/editor/owl/model/entity/RandomProlong.java

```diff
@@ -0,0 +1,53 @@
+package org.protege.editor.owl.model.entity;
+
+import org.semanticweb.owlapi.model.OWLEntity;
+
+import uk.org.russet.identitas.*;
+
+public class RandomProlong extends AbstractIDGenerator implements AutoIDGenerator, Revertable {
+
+	
+	private String nextId;
+    public RandomProlong () {
+    	try {
+			nextId = Util.randomProlong();
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+    }
+	
+    public String getNextID(Class<? extends OWLEntity> type) throws AutoIDException {
+        String id = nextId;
+        try {
+			nextId = Util.randomProlong();
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+        
+        return getPrefix(type) + id.toString().replace("-", "_") + getSuffix(type);
+        
```

Comment:
@Nizal-Shammry why replace the dashes with underscored here?  Please can you provide a brief comment about this?  Both in the review and in the code.  Thanks.